store.json

```diff
@@ -0,0 +1,4 @@
+[{
```

Comment:
This store.json should probably be gitignored? Would suggest we make a data directory and gitignore that instead of writing to project root